python/grass/temporal/core.py

```diff
@@ -363,8 +363,8 @@ def get_raise_on_error():
 
 
 def get_tgis_version():
-    """Get the version number of the temporal framework
-    :returns: The version number of the temporal framework as string
+    """Get the supported version number of the temporal framework
+    :returns: The version number of the temporal framework as integer
```

Comment:
I must say I don't understand the role of tgis_version and how it relates to tgis_db_version, specifically why it is stored in the database, so I can't comment much on this part.

Reply:
https://github.com/OSGeo/grass/blob/master/python/grass/temporal/core.py#L116
https://github.com/OSGeo/grass/blob/master/python/grass/temporal/core.py#L365